collectors/tc.plugin/metadata.yaml

```diff
@@ -36,7 +36,12 @@ modules:
           description: ""
     setup:
       prerequisites:
-        list: []
+        list:
+          - title : command
+            description: |
+              It is necessary to have command `tc` installed on host, and it is suggested to install [FireQOS](http://firehol.org/tutorial/fireqos-new-user/).
```

Comment:
```suggestion
              It is necessary to have the command `tc` installed on host, and it is suggested to install [FireQOS](http://firehol.org/tutorial/fireqos-new-user/).
```

Reply:
- I am not sure about installing `tc`. I think any basic system has it.
- If the above is wrong and you need to install it then it should be 2 separate steps: install `tc`, install `FireQOS`.

Reply:
The title definitely shouldn't be `command` - it must describe the content (e.g. Install FireQOS).

Reply:
I removed FireQOS, as this is not something you need to get the collector up and running, the README states: `If you also use [FireQOS](http://firehol.org/tutorial/fireqos-new-user/) it will collect interface and class names.`